modules/apps/portal/portal-upgrade-impl/src/main/java/com/liferay/portal/upgrade/internal/report/UpgradeReport.java

```diff
@@ -121,6 +130,112 @@ private String _getDialectInfo() {
 			StringPool.PERIOD, db.getMinorVersion(), StringPool.NEW_LINE);
 	}
 
+	private String _getDocLibSize() {
+		if (_dlStore.equals("com.liferay.portal.store.db.DBStore")) {
+			return StringPool.BLANK;
+		}
+		else if (_dlStore.contains("FileSystemStore")) {
+			if ((_rootDir == null) &&
+				_dlStore.equals(
+					"com.liferay.portal.store.file.system." +
+						"AdvancedFileSystemStore")) {
+
+				return StringBundler.concat(
+					"\"rootDir\" was not set, unable to determine the size of ",
+					"the document library", StringPool.NEW_LINE);
+			}
+
+			final AtomicLong length = new AtomicLong(0);
+
+			String docLibPath =
+				PropsValues.LIFERAY_HOME + "/data/document_library";
+
+			if (_rootDir != null) {
+				docLibPath = _rootDir + "/data/document_library";
```

Comment:
is this necessary? I mean, when we set the rootDir via configAdmin I believe (not totally sure) that that's the final folder and adding _/data/document_library_ is not needed.

Reply:
You are correct, appending /data/document_library is not needed and makes it point to the wrong directory (one that won't exist) I'll add this change on a commit and send to you.